### ISSUE TITLE
USER ISSUE: bison bug

### ISSUE THREAD

**Issue by pbplugge** [StrangeLoopGames/EcoIssues#11578]
**Version:** 0.8.0.5 beta

**Steps to Reproduce:**
kill a bison.
can not pick up bison.
bison disappears.
when getting back to base tent disappears.
reconnect, tent is back.

**Expected behavior:**
none of the above

**Actual behavior:**

**Do you have mods installed? Does issue happen when no mods are installed?:**

**Comment by SeanAtr0n** [comment]
has this happened for you again? was just your tent gone or the ground around it as well?

**Comment by pbplugge** [comment]
It happened to me once with the bison and the tent. Only the tent disappeared. I think you may have an array of objects and when one item gets deleted something goes wrong while re-sorting the array or overwriting a deleted item with the last item of the array. Good luck!
With wood it happens all the time. Chunks of wood disappear when chopping but you find out after reconnecting. It also can become un-hittable.

**Comment by D3nnis3n** [comment]
Is this issue still happening?

**Comment by pbplugge** [comment]
I did not experience this issue again. only once.

**Comment by D3nnis3n** [comment]
Alright, i'm closing this as not reproducable then. If it ever occurs again and you find a reproduction case, feel free to open a new issue.
